### PR TITLE
Update contributors and changelog for 1.7.6.1

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Release 1.7.6.1 changelog
| Type?         | improvement
| Category?     | ME
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | nothing fixed
| How to test?  | nothing to test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15264)

### CODE REVIEW

File: docs/CHANGELOG.txt [hunk 1]
```diff
@@ -24,6 +24,57 @@ International Registered Trademark & Property of PrestaShop SA
 Release Notes for PrestaShop 1.7
 --------------------------------
 
+####################################
+#   v1.7.6.1 - (2019-08-23)
```
Comment (matks) [inline docs/CHANGELOG.txt]:
```suggestion
#   v1.7.6.1 - (2019-08-26)
```
🤔 ?

File: docs/CHANGELOG.txt [hunk 2]
```diff
@@ -24,6 +24,57 @@ International Registered Trademark & Property of PrestaShop SA
...
+    - #14711: Update Decimal library to 1.2.0 (by @eternoendless)
+    - #14833: Bad currency format for Czech language (by @PierreRambaud)
+    - #14664: Update precision and numeric iso code of currencies (by @Quetzacoalt91)
+- Installer:
+  - Bug fix:
+    - #14620: Init properly the Kernel on major *manual* upgrades (by @Quetzacoalt91)
+    - #14739: Keep Composer.lock file in release (by @jolelievre)
+- Web Services:
+  - Bug fix:
+    - #14953: Initialize container in WebService environment (Progi1984's work on 1.7.7) (by @matthieu-rolland)
```
Comment (matks) [inline docs/CHANGELOG.txt]:
```suggestion
    - #14953: Initialize container in WebService environment (by @matthieu-rolland)
```

Comment (matthieu-rolland) [reply docs/CHANGELOG.txt]: I implemented your suggestion @matks , it's because I prepared the release on Friday :D

**Review by matks — DISMISSED** [review]

File: w3m [hunk 3]
```diff
@@ -0,0 +1,116 @@
+<h1 id="cldr-remove-number-of-decimals-in-shop-parameter-general">[CLDR] Remove Number of decimals in Shop Parameter &gt; General</h1>
```
Comment (eternoendless) [inline w3m]: I think this file doesn't belong here :)

**Review by eternoendless — CHANGES_REQUESTED** [review]

File: w3m (hunk 3 above)
Comment (matthieu-rolland) [reply w3m]: actually there was 2 wrong files 🤦‍♂   I fixed it thanx !

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @matthieu-rolland

**Pull request merged**